modules/gdscript/tests/scripts/analyzer/warnings/get_node_without_onready.gd

```diff
@@ -8,6 +8,12 @@ var without_self = get_node(^"Node")
 var with_cast = get_node(^"Node") as Node
 var shorthand_with_cast = $Node as Node
 
+var shorthand_unique = %UniqueNode
+var with_self_unique = self.get_node(^"%UniqueNode")
+var without_self_unique = get_node(^"%UniqueNode")
+var with_cast_unique = get_node(^"%UniqueNode")
+var shorthand_with_cast_unique = %UniqueNode as Node
```

Comment:
I'm not sure if we should duplicate all the cases, but we should add a test for `$"%UniqueNode"`.